styleguides/html.md

```diff
@@ -0,0 +1,38 @@
+_This styleguide has been adapted from
+https://google.github.io/styleguide/htmlcssguide.xml._
+
+-------------------------------------------------------------------------------
+
+# HTML Styleguide
+
+This is a collection of HTML guidelines that Hack Club workshops should follow
```

Comment:
@paked I think having `should` is important because there may be situations that these guides can't be followed.